candle_stick_chart.ipynb

```diff
@@ -0,0 +1,227 @@
N9+/ZG06ZNje3btxuGYRgOh8NwOp3GmDFjjLCwMOOHH34oqi4XG5eqXePGjY2dO3cahmEYBw4ccM13Op2F3s/ipqDPOafTaaxevdoYPHiwUatWLeP7778vqi4XC8eOHTNuv/1245lnnnFNS0tLM1q2bGlMnjzZbdm9e/caH3zwgTF8+HDjH//4h7Fr167C7m6x4UndztWvXz9j9+7dhdHFYsvT2u3du9cYMmQIr1cUKkYyi7GjR4/qyJEjrsO22dnZCgkJ0ccff6ygoCANHTpUUt5VljabTffcc482bNigBg0aFGW3i4VL1a5MmTJ6/PHHJcltxLIknw+Xr6DPufwLWsqVK6dVq1aV+ENvaWlpKl++vOsQuNPpVHh4uNq1a+c6Hzg7O1uSVL16dXXr1k2TJ0/WE088oejo6CLrd1ErSN3OPY0gNzdXkjR//nzVrl278DtcjHhau/Lly+vOO+/UJ598UuJfryg8hMxirEOHDsrJydFLL70kKe/2RdnZ2fL19dWcOXO0b98+zZs3T1LeH9/Y2FgC5h8uV7v9+/e7aoc/FeQ5N3fuXElSmzZtNHr0aNWpU6cou1wshIeHq1evXmrTpo2kP/9hcTgcrovvzj3NAHkKUrdzTwnKv3APnteuTJkyatOmjWrUqFHofUXJRcgspgzDULly5fTYY4/p448/1vz58yXlvekbhqFKlSrpuuuu0+HDhyXxx/dcntYOeQpat9TUVNc6/v7+RdTb4qVcuXLq3LmzpLw65r/h+/r6ukbfbDabXn75Zb344otF1s/ihrqZR+1wNSCZFFM2m01+fn7q3r279u7dq3fffVdnz57VI488IpvNpuDgYNcV5pL7H5mSjtqZQ92scW5NypYt6xrBfP755/Xmm29q9erVRdW1Yo26mUftUFzZDIMbjRU3f33z3rVrl2bPnq01a9YoPj5eN954o3bs2KElS5Zo/fr1Jf7cpHNRO3Oom3mXCtsvvfSSDh48qFq1amny5Mlas2aNGjZsWMg9LJ6om3nUDlcLRjKLgfw/GOnp6QoPDz/vj0edOnU0YsQItWnTRq+99pp+++03BQUFadWqVSX+zZ7amUPdzLtc7c6Vm5urRYsWqXTp0vr4449L9Js9dTOP2uGqVViXsePSvvnmG6NXr15ut465GIfDYZw+fboQenV1oHbmUDfzClq7hQsXGvHx8a5bP5V01M08aoerERf+FBO+vr5au3at6ya6F5L/Wdo+Pj4KDAwsrK4Ve9TOHOpmXkFqJ0k333yzli9frvr16xdSz4o36mYetcPViJBZhIw/Tod1OBy6/vrr9dRTT+mf//ynfvvttwsuX5I/S/uvqJ051M08T2snSZUrV1ZkZGRhdbFYom7mUTtc7XgHKUKnTp2S9OcbeXx8vA4dOqRffvlF0p+jSDgftTOHuplH7cyhbuZRO1ztCJlF5LvvvlO3bt20ZMkSnThxQpLUuXNn1a9fX6NHj1Zubi6jSBdB7cyhbuZRO3Oom3nUDtcCnqGFLP/wx7Fjx9S4cWM99dRTGjRokF555RVJ0pAhQ1S2bFktX77cbXlQO7Oom3nUzhzqZh61w7WE+2QWEuOPW1Dk5ua6fTrP119/rbVr12rRokWqXLmy2rZtq48++kgtWrTQ1KlTi7DHxQe1M4e6mUftzKFu5lE7XIsImYUg/4/Hhg0btGLFCp04cUJ169ZV3759FR4eruzsbJ0+fVovvPCCjh07pg8++EB2u10pKSkKCQkp0Z+qQu3MoW7mUTtzqJt51A7XKkJmIVm5cqUGDBigu+66SwcOHNCJEyd05MgRrV69Wtddd51rucOHD+vTTz/VDTfcwC0o/kDtzKFu5lE7c6ibedQO1yTv3oYThmEYR44cMVq3bm1MmzbNNW379u1G9+7djUaNGhXoZtglFbUzh7qZR+3MoW7mUTtcq7jwxwtOnjwp6c/bS5w8eVKpqalq0KCBa5mYmBg999xzCg0N1dKlSyVxArdE7cyibuZRO3Oom3nUDiUFIdNi6enpatiwof773/+6bi8RERGh6667Tps3b3YtZ7fbFRcXJ19fX+3atUuSSvx5NdTOHOpmHrUzh7qZR+1QkhAyLebj46OEhAQ9+uijWrVqlWtakyZNtHHjRtdtJ6S8PxiVKlVSaGioDMMo8f+lUjtzqJt51M4c6mYetUNJwoU/XpCenq5XX31Vb7/9tv71r3+pa9euOnbsmB5++GFlZmaqSZMmat68ub744gstWrRIn376qerUqVPU3S4WqJ051M08amcOdTOP2qHEKNxTQK9NWVlZRkZGhtu0Q4cOGSNGjDDCwsKMDz/80DAMwzh69Kjx97//3ejYsaPRpEkTIyEhwfjhhx+KosvFBrUzh7qZR+3MoW7mUTuUVITMK5SSkmK0bt3a+L//+z9j4cKFxrp161zzzpw5YwwfPtwICwsz/vvf/xqGYRg5OTlGbm6ukZ6ebmRlZRVRr4sHamcOdTOP2plD3cyjdijJfC8/1omLcTqdWrBggZKTkxUYGKjXX39dp0+fVlhYmG644Qb16dNH9957r8qVK6cHHnhAZcqUUfv27SVJFSpUKOLeFy1qZw51M4/amUPdzKN2KOk4J/MKpaamatq0adq7d69q1qypAQMGaPHixfryyy+1fft2hYWFqVq1avr++++Vnp6u5cuXq1WrVkXd7WKB2plD3cyjduZQN/OoHUoyRjKvUGRkpAYPHqypU6fqf//7n2rWrKmnn35akrR161YdPnxY8+fPV4UKFZSenq7y5csXcY+LD2pnDnUzj9qZQ93Mo3YoyRjJtMjhw4c1depUffPNN+rSpYuGDRvmmpeTkyOn06nMzEyFh4cXYS+LJ2pnDnUzj9qZQ93Mo3YoiQiZFkpNTdWrr76q7777Tl26dNHQoUMlSbm5ufL1ZdD4UqidOdTNPGpnDnUzj9qhpCFkWiz/j8gPP/ygW265Rc8880xRd+mqQe3MoW7mUTtzqJt51A4lCZ
```

Comment:
I am not sure that we want to keep this in git. What does this file correspond to?

Reply:
This is a notebook with example candle stick charts in it. Basically it's a test.